contrib/zenith/libpagestore.c

```diff
@@ -124,14 +122,14 @@ zenith_connect()
 				PQfinish(pageserver_conn);
 				pageserver_conn = NULL;
 
-				zenith_log(ERROR, "[ZENITH_SMGR] failed to get handshake from pageserver: %s",
-						   msg);
+				neon_log(ERROR, "could not complete handshake with pageserver: %s",
+						 msg);
 			}
 		}
 	}
 
-	// FIXME: when auth is enabled this ptints JWT to logs
-	zenith_log(LOG, "libpqpagestore: connected to '%s'", page_server_connstring);
+	/* FIXME: when auth is enabled this prints JWT to logs */
+	neon_log(LOG, "libpagestore: connected to '%s'", page_server_connstring);
```

Comment:
while on it

```suggestion
	neon_log(LOG, "libpagestore: connected to '%s'", page_server_connstring_raw);
```

This will print `[NEON_SMGR] libpagestore: connected to 'postgresql://no_user:$ZENITH_AUTH_TOKEN@localhost:15008'` instead of actual JWT